### PR TITLE
Update tracker-surrogates to namespaced version

### PR DESCRIPTION
**Pull request by jonathanKingston**
**Reviewer:** @kdzwinel 

## Description:

Update to: https://github.com/duckduckgo/tracker-surrogates/pull/3

https://app.asana.com/0/0/1202795530517779/f

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

**Comment by jonathanKingston** [comment]
The test fail is unrelated and fixed in: https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/1342

File: package-lock.json [hunk 1]
```diff
@@ -16341,7 +16340,7 @@
         "@duckduckgo/autofill": {
             "version": "git+ssh://git@github.com/duckduckgo/duckduckgo-autofill.git#d29c5abc7f473b69f3d81d8a15e137ed3aab029d",
             "integrity": "sha512-RrmmFbGxMV4aKYA2ebJ6PU0IH9CW57iIYnvBemj9ZUGe8DlAheaBTOPSH3AQQk0aY7SFhNFc8UA4wBT26uM3IQ==",
-            "from": "@duckduckgo/autofill@github:duckduckgo/duckduckgo-autofill#d29c5abc7f473b69f3d81d8a15e137ed3aab029d",
```
Comment (kdzwinel) [inline package-lock.json:16344]: hm, this seems random?

Comment (jonathanKingston) [reply package-lock.json:16344]: Yeah I've noticed this.

I think it's a difference with local edits of npm vs dependabot; we have had similar before but they actually caused issues.

The integrity here is the same and this commit points to the #5.0.1 so means the same thing.

**Review by kdzwinel — APPROVED** [review]
LGTM thanks!

**Pull request merged**